rest/server_context.go

```diff
@@ -411,8 +411,8 @@ func (sc *ServerContext) _getOrAddDatabaseFromConfig(config *DbConfig, useExisti
 
 	bucket, err := db.ConnectToBucket(spec, func(bucket string, err error) {
 
-		msg := fmt.Sprintf("%v dropped Mutation feed (TAP/DCP) due to error: %v, taking offline", bucket, err)
-		base.Warn(msg)
+		msg := fmt.Sprintf("%v dropped Mutation feed (TAP/DCP) due to error: %v, taking offline", base.UD(bucket), err)
```

Comment:
We'll have to move the format + args of `Sprintf` into the `base.WarnR` call for it to be redacted correctly.

Reply:
Fixed this in a somewhat awkward way, where in one case where it passes the `reason` string, it just says "Bucket" rather than the actual bucket name